README.md

```diff
@@ -298,13 +298,14 @@ Following folder are created with these user rights:
 
 Here's the policy we follow for tagging our Docker images:
 
-| Tag            | Support level | Docker base image |
-|:--------------|:-------------|---|
-| latest         | <ul><li>latest JanusGraph release</li><li>no breaking changes guarantees</li></ul> | openjdk:8-jre-slim-buster |
-| 0.x            | <ul><li>newest patch-level version of JanusGraph</li><li>expect breaking changes</li></ul> | openjdk:8-jre-slim-buster |
-| 0.x.x          | <ul><li>defined JanusGraph version</li><li>breaking changes are only in this repo</li></ul> | openjdk:8-jre-slim-buster |
-| 0.x.x-revision | <ul><li>defined JanusGraph version</li><li>defined commit in JanusGraph-docker repo</li></ul> | openjdk:8-jre-slim-buster |
-
+| Tag            | Support level | Docker base image      |
+|:--------------|:-------------|------------------------|
+| latest         | <ul><li>latest JanusGraph release</li><li>no breaking changes guarantees</li></ul> | eclipse-temurin:11-jdk |
+| x.x            | <ul><li>newest patch-level version of JanusGraph</li><li>expect breaking changes</li></ul> | eclipse-temurin:8-jdk  |
+| x.x.x          | <ul><li>defined JanusGraph version</li><li>breaking changes are only in this repo</li></ul> | eclipse-temurin:8-jdk  |
+| x.x.x-revision | <ul><li>defined JanusGraph version</li><li>defined commit in JanusGraph-docker repo</li></ul> | eclipse-temurin:8-jdk  |
+
+Note: we start to use `eclipse-temurin:11-jdk` as base image since 1.0 version.
```

Comment:
Do you want to highlight the switch to `eclipse-temurin` or the use of JDK 11 here?
If it's the first, then it's misleading to say _since 1.0 version_  as this also applies to the `0.5` and `0.6` images.
If it's the latter, , then that's already included in the CHANGELOG, linked right below.

Reply:
I intended to highlight the use of JDK 11 here but you are right, this might create more confusion. I'll remove it.